connectorx/src/destinations/arrow/arrow_assoc.rs

```diff
@@ -71,6 +72,51 @@ impl_arrow_assoc!(f32, ArrowDataType::Float32, Float32Builder);
 impl_arrow_assoc!(f64, ArrowDataType::Float64, Float64Builder);
 impl_arrow_assoc!(bool, ArrowDataType::Boolean, BooleanBuilder);
 
+const DEFAULT_ARROW_DECIMAL_PRECISION: u8 = 38;
```

Comment:
Move these constants to `connectorx/constants.rs` and share them between `arrow` and `arrowstream`?